katemalone/rancid-tomatillos
# Login and app testing

#### What's this PR do?

This PR adds tests for apiCalls, App, and LoginForm.  It includes async tests, but is missing the handleSubmit test on loginForm

#### Where should the reviewer start?

You can start anywhere, just the test files are relevant.

#### How should this be manually tested?

It might work to pull down and make sure the tests are running on your machine as well.  All should currently be passing.

#### Any background context you want to provide?

Super sorry I merged my master branch up - this contains everything that was in there before, so I don't think anything has been lost.

#### What are the relevant tickets?
https://github.com/katemalone/rancid-tomatillos/issues/20
https://github.com/katemalone/rancid-tomatillos/issues/18

#### Screenshots (if appropriate)

<img width="570" alt="Screen Shot 2019-12-18 at 11 14 56 PM" src="https://user-images.githubusercontent.com/24462031/71155288-9b580900-21fa-11ea-91f1-9d5ffb6b1e7d.png">
